gherkin/javascript/src/Errors.ts

```diff
@@ -20,9 +20,10 @@ class GherkinException extends Error {
     }
   }
 
-  protected static _create<T>(location: messages.ILocation, message: string) {
-    const column = location.column || 0
-    const m = `(${location.line}:${column}): ${message}`
+  protected static _create<T>(message: string, location?: messages.ILocation) {
```

Comment:
@vincent-psarga I flipped the argument order of some methods. But I can't really judge of that would be a breaking change or not. I don't think this class is intended to be used by the public?